test-harness/template.yaml

```diff
@@ -0,0 +1,373 @@
+Transform: "AWS::Serverless-2016-10-31"
+Description: "Digital Identity IPV Return Test Automation Harness"
+
+Parameters:
+  CodeSigningConfigArn:
+    Type: String
+    Default: "none"
+    Description: >
+      The ARN of the Code Signing Config to use, provided by the deployment pipeline
+  Environment:
+    Description: "The environment type"
+    Type: "String"
+    AllowedValues:
+      - "dev"
+      - "build"
+  PermissionsBoundary:
+    Description: "The ARN of the permissions boundary to apply when creating IAM roles"
+    Type: String
+    Default: "none"
+  VpcStackName:
+    Type: String
+    Default: "vpc-cri"
+    Description: The name of the VPC stack deployed.
+  BackendStack:
+    Description: The stack name of the API stack under test
+    Default: ipvreturn-api
+    Type: String
+
+Mappings:
+  EnvironmentConfiguration: # This is where you store per-environment settings.
+    dev:
+      logretentionindays: 3
+      apiTracingEnabled: true
+    build:
+      logretentionindays: 3
+      apiTracingEnabled: true
+
+  PlatformConfiguration:
+    dev:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+    build:
+      CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+
+  EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
+    dev:
+      TESTHARNESSURL: "testharness.return.dev.account.gov.uk"
+    build:
+      TESTHARNESSURL: "testharness.return.build.account.gov.uk"
```

Comment:
this does not match the main template TESTHARNESS spec

Reply:
Main template TESTHARNESS spec has been updated to match test-harness template